### DISCUSSION TITLE
v2.0.0-beta.55  这种变动太大了吧

### DISCUSSION THREAD

**Discussion by yhjyang1** [alibaba/formily#1477] [Ideas]
![image](https://user-images.githubusercontent.com/37994732/119104538-f674d780-ba4e-11eb-96d1-2fb366607bcc.png)
![image](https://user-images.githubusercontent.com/37994732/119104579-fecd1280-ba4e-11eb-84b5-acc2526a987d.png)
打包的文件怎么变化这么大

**Comment by janryWang** [comment]
具体什么问题

**Reply by yhjyang1** [reply]
之前用的时候是能用 `import * from '@formily/antd/esm/submit'`的，今天 我看node_modules里面都没有esm文件夹了，不知道类似这样的变动还有多少

**Reply by janryWang** [reply]
构建出问题了，马上修复

**Reply by janryWang** [reply]
刚刚在搞workflow这块的优化，没想到对这个有影响，很不好意思

**Comment by liuweiGL** [comment]
能问下为什么不使用 `import {submit} from @formily/antd` 的方式导入吗？你有遇到 #1482 问题不？

**Reply by yhjyang1** [reply]
@formily/antd 在第一行把全部的style文件打进去了，这样我要是想override antprefix的话就不生效，只能单独引用
